R/loadComp.R

```diff
@@ -380,6 +382,12 @@ predictSolute.loadComp <- function(
     }
   }
   
+  #use aggregate solute to aggregate to agg.by, but warn and return NA for uncertainty
+  if(agg.by != "unit") {
+    preds <- aggregateSolute(preds, metadata = getMetadata(load.model), agg.by = agg.by,
+                             format = flux.or.conc)
```

Comment:
@aappling-usgs `aggregateSolute` has both `flux rate` and `flux total` options, which of those is appropriate for these?  Should one of them be eliminated?